site-src/blog/2023/0829-mesh-support.md

```diff
@@ -0,0 +1,210 @@
+
+CEL validation is not supported, but Gateway API v0.8.0 CRDs can still be
+installed. When you upgrade to Kubernetes 1.25+, the validation included in
+these CRDs will automatically take effect.
+
+We recommend continuing to use the validating webhook on these Kubernetes
+versions.
+
+#### Kubernetes 1.22 and older
+
+Gateway API only commits to support for [5 most recent versions of
+Kubernetes][supported-versions]. As such, these versions are no longer
+supported by Gateway API, and unfortunately Gateway API v0.8.0 cannot be
+installed on them, since CRDs containing CEL validation will be rejected.
+
+### GEP Process Changes
+
+The second significant change in Gateway API v0.8.0 is that we have (by
+necessity) taken a hard look at the process around [Experimental][status]
+GEPs. Some of these GEPs have been lingering long enough that projects have
+come to rely on them in production use, which is a bit of a breakdown of the
+GEP process. In order to prevent it happening in the future, we have changed
+the GEP process such that reaching [Experimental][status] _requires_ that a
+GEP include both the graduation criteria by which the GEP will become
+[Standard][status], and a probationary period after which the GEP will be
+dropped if does not meet its graduation criteria.
+
+For an exhaustive list of changes included in the `v0.8.0` release, please see
+the [v0.8.0 release notes]. For more information on Gateway API versioning,
+refer to the [official documentation][versioning docs].
+
+## How can I get started with the Gateway API?
+
+Gateway API represents the future of load balancing, routing, and service mesh
+APIs in Kubernetes. There are already more than 20 [implementations][impl]
+available (including both ingress controllers and service meshes) and the list
+keeps growing.
+
+If you're interested in getting started with Gateway API, take a look at the
+[API concepts documentation][concepts] and check out some of the
+[Guides][guides] to try it out. Because this is a CRD-based API, you can
+install the latest version on any Kubernetes 1.23+ cluster.
+
+If you're specifically interested in helping to contribute to Gateway API, we
+would love to have you! Please feel free to [open a new issue][issue] on the
+repository, or join in the [discussions][disc]. Also check out the [community
+page][community] which includes links to the Slack channel and community
+meetings. We look forward to seeing you!!
+
+## Further Reading:
+
+- [GEP-1324] provides an overview of the GAMMA goals and some important
+  definitions. This GEP is well worth a read for its discussion of the problem
+  space.
+- [GEP-1426] defines how to use Gateway API route resources, such as
+  HTTPRoute, to manage traffic within a service mesh.
+- [GEP-1686] builds on the work of [GEP-1709] to define a _conformance
+  profile_ for service meshes to be declared conformant with the Gateway API.
+
+Although these are [Experimental][status] APIs, note that they are available
```

Comment:
I think it's very important to clarify that these are just patterns to use with existing APIs, not APIs themselves.
```suggestion
Although these are [Experimental][status] patterns, note that they are available
```

Reply:
Yeah, good point.